src/Circlereader/util/optionsAttribute.ts

```diff
@@ -0,0 +1,636 @@
+import {LSHProjectionOptions} from '../circle-analysis/circle/l-s-h-projection-options';
+import {LSHProjectionType} from '../circle-analysis/circle/l-s-h-projection-type';
+import {LSTMKernelType} from '../circle-analysis/circle/l-s-t-m-kernel-type';
+import {LSTMOptions} from '../circle-analysis/circle/l-s-t-m-options';
+import {L2NormOptions} from '../circle-analysis/circle/l2-norm-options';
+import {LeakyReluOptions} from '../circle-analysis/circle/leaky-relu-options';
+import {LocalResponseNormalizationOptions} from '../circle-analysis/circle/local-response-normalization-options';
+import {MirrorPadMode} from '../circle-analysis/circle/mirror-pad-mode';
+import {MirrorPadOptions} from '../circle-analysis/circle/mirror-pad-options';
+import {OneHotOptions} from '../circle-analysis/circle/one-hot-options';
+import {Operator} from '../circle-analysis/circle/operator';
+import {PackOptions} from '../circle-analysis/circle/pack-options';
+import {Padding} from '../circle-analysis/circle/padding';
+import {Pool2DOptions} from '../circle-analysis/circle/pool2-d-options';
+import {RNNOptions} from '../circle-analysis/circle/r-n-n-options';
+import {ReducerOptions} from '../circle-analysis/circle/reducer-options';
+import {ReshapeOptions} from '../circle-analysis/circle/reshape-options';
+import {ResizeBilinearOptions} from '../circle-analysis/circle/resize-bilinear-options';
+import {ResizeNearestNeighborOptions} from '../circle-analysis/circle/resize-nearest-neighbor-options';
+import {ReverseSequenceOptions} from '../circle-analysis/circle/reverse-sequence-options';
+import {SVDFOptions} from '../circle-analysis/circle/s-v-d-f-options';
+import {SequenceRNNOptions} from '../circle-analysis/circle/sequence-r-n-n-options';
+import {ShapeOptions} from '../circle-analysis/circle/shape-options';
+import {SkipGramOptions} from '../circle-analysis/circle/skip-gram-options';
+import {SoftmaxOptions} from '../circle-analysis/circle/softmax-options';
+import {SpaceToDepthOptions} from '../circle-analysis/circle/space-to-depth-options';
+import {SparseToDenseOptions} from '../circle-analysis/circle/sparse-to-dense-options';
+import {SplitOptions} from '../circle-analysis/circle/split-options';
+import {SqueezeOptions} from '../circle-analysis/circle/squeeze-options';
+import {StridedSliceOptions} from '../circle-analysis/circle/strided-slice-options';
+import {SubOptions} from '../circle-analysis/circle/sub-options';
+import {TensorType} from '../circle-analysis/circle/tensor-type';
+import {TransposeConvOptions} from '../circle-analysis/circle/transpose-conv-options';
+import {UnidirectionalSequenceLSTMOptions} from '../circle-analysis/circle/unidirectional-sequence-l-s-t-m-options';
+import {UniqueOptions} from '../circle-analysis/circle/unique-options';
+import {UnpackOptions} from '../circle-analysis/circle/unpack-options';
+import {WhileOptions} from '../circle-analysis/circle/while-options';
+import {NodeAttributes} from '../type/types';
+
```

Comment:
Have you checked to put
```
/* eslint-enable @typescript-eslint/naming-convention */
```
here? does the below codes have warnings ?

Reply:
yes. This is also required in the code below.

Reply:
Q) naming convention may ask to start with lower case and camel case. Am I correct?
If so, would it be a problem to use like `cirNodeAttributes` for `NodeAttributes` ?

Reply:
In general, it is correct to use a `camel case`, but since the name is declared as a `type`, it is written in an `upper camel case` (`Pascal case`) to distinguish it from a variable name. If there is a problem with this part of the code rules, We will fix it.

Reply:
Hmm... @seanshpark 's question makes sense.

i.e., we already declare `Conv2DOption` as `let conv2DOpt = new Conv2DOptions();`.
Then, we may be able to declare `SVDFOption` as let `svdfOpt = new SVDFOptions();` and we can put enabling eslint at this line.

@yesfordev Could you check whether it works? :)